[BUG] Local tests fail when run repeatedly

Local `pytest` once will pass.  
Local `pytest` a second time will fail with:
```
FAILED recOrder/tests/util_tests/test_create_empty.py::test_create_empty_hcs_zarr - assert (1, 3, 1, 1024, 1024) == (1, 2, 1, 1024, 1024)
```

The `test_store.zarr` is not being handled correctly. After `rm -r test_store.zarr`, the tests will pass again.

--- a/recOrder/tests/util_tests/test_create_empty.py
+++ b/recOrder/tests/util_tests/test_create_empty.py
@@ -6,8 +6,8 @@ from iohub.ngff import Position, open_ome_zarr
 from recOrder.cli.utils import create_empty_hcs_zarr
 
 
-def test_create_empty_hcs_zarr():
-    store_path = Path("test_store.zarr")
+def test_create_empty_hcs_zarr(tmp_path):
+    store_path = tmp_path / Path("test_store.zarr")
     position_keys: list[tuple[str]] = [
         ("A", "0", "3"),
         ("B", "10", "4"),

